src/pymovements/dataset/dataset_definition.py

```diff
@@ -39,6 +40,67 @@
 yaml.add_multi_constructor('!', type_constructor, Loader=yaml.SafeLoader)
 
 
+# sphinx automatically substitutes _Resources with the Union using the | operator.
+_Resources = Union[dict[str, list[dict[str, str]]], dict[str, tuple[dict[str, str], ...]]]
```

Comment:
why is this not possible?

`dict[str, list[dict[str, str]]] | dict[str, tuple[dict[str, str], ...]]`

Reply:
this is just a `TypeAlias`, right?

this works for tests and pre-commit:

`_Resources: TypeAlias = dict[str, list[dict[str, str]]] | dict[str, tuple[dict[str, str], ...]]`

Reply:
Now I remember why it didn't work out: `|` is not a valid operator in python 3.9. I'm really looking forward to drop py39 support.

Reply:
@SiQube unfortunately this was not possible (see the workflows of my last two commits), but will be possible for python310+.

Can you approve if there aren't any more change requests?

Reply:
@SiQube just a small reminder that this PR was already reviewed and needs a final approve.

Reply:
@SiQube small reminder that this PR was already reviewed and just needs your final approve.